### PR TITLE
Issue 45006: Provide link to lookup table/query in field designer

### PR DESCRIPTION
**Pull request by RosalineP**
#### Rationale
Usability improvement garnered by turning the lookup text display on a domain designer field into a direct link to the query.

#### Related Pull Requests
* https://github.com/LabKey/labkey-ui-components/pull/869
* https://github.com/LabKey/platform/pull/3458
* https://github.com/LabKey/biologics/pull/1386
* https://github.com/LabKey/sampleManagement/pull/1013

#### Changes
* Bump version

### CODE REVIEW

**Comment by RosalineP** [comment]
@cnathe 
I'm a little stale on the deal with current frontend package updates—should I update, for good measure, other repos as well with this change?

**Comment by cnathe** [comment]
> @cnathe I'm a little stale on the deal with current frontend package updates—should I update, for good measure, other repos as well with this change?

@RosalineP In this case since the field editor is used in the LKB and LKSM app UI, I would suggest that you bump the @labkey/components version for those two via the `biologics/package.json` and the `sampleManagement/package.json` so that TeamCity runs those suites to check for regressions.

File: core/package-lock.json [hunk 1]
```diff
@@ -32,7 +32,6 @@
       "version": "2.1.2",
       "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.1.2.tgz",
       "integrity": "sha512-hoyByceqwKirw7w3Z7gnIIZC3Wx3J484Y3L/cMpXFbr7d9ZQj2mODrirNzcJa+SM3UlpWXYvKV4RlRpFXlWgXg==",
-      "dev": true,
```
Comment (cnathe) [inline core/package-lock.json]: hmmm...did you run the npm install command with the legacy peer deps flag (i.e. `npm install --legacy-peer-deps`)? I would not have expected this many line changes to the package-lock.json file with just a components package bump.

**Review by cnathe — CHANGES_REQUESTED** [review]

File: core/package-lock.json (hunk 1 above)
Comment (RosalineP) [reply core/package-lock.json]: Ah, I think I didn't! The package-lock is looking better now.

Branches also opened on LKB and LKSM!

**Review by cnathe — APPROVED** [review]

**Pull request merged**